lib/postgres/postgres-pr/connection.rb

```diff
@@ -121,11 +121,11 @@ def initialize(database, user, password=nil, uri = nil)
     end
   end
 
-  def address
+  def peeraddress
```

Comment:
```suggestion
  def peerhost
```
Think this is meant to be `peerhost` I don't see any mention of `peeraddress` in the code base